### ISSUE TITLE
Not Working* in DSA Self Paced course Problems

### ISSUE THREAD

**Issue by Vaibhavsingh3477** [AtharvaNanavate/GfG-To-GitHub#9]
Not Working in DSA Self Paced course Problems

**Comment by Prateek-Wayne** [comment]
Same for me also.
It is not working if I solve problem by going in the DSA Self Paced Problem
But If I Submit Solution for the same problem by picking the problem from the problems sections then it is working perfectly.

**Comment by AtharvaNanavate** [comment]
Yes, currently, only problems with 'https://www.geeksforgeeks.org/problems' or 'https://practice.geeksforgeeks.org/problems' in the URL are supported. Adding support for problems within courses adds complexity, as the structure of problem pages differs within courses. 

Thanks for raising this issue, it may be considered for future development.

**Comment by Vaibhavsingh3477** [comment]
Thank you for your response and support. And for resolving the issue in my
mind.

**Comment by pranavghadge** [comment]
yeah it doesn't work with the courses we have taken from GFG like when we under stand a topic and solve related questions given within the course
